libtransmission/torrent.cc

```diff
@@ -1770,29 +1770,15 @@ void tr_torrent::create_empty_files() const
         return;
     }
 
-    auto paths = std::array<std::string_view, 4>{};
-    auto const n_paths = location_helpers::buildSearchPathArray(this, std::data(paths));
-    if (n_paths == 0)
-    {
-        return;
-    }
-    auto const base = paths[0];
-
-    auto file_count = this->file_count();
+    auto const base = download_dir();
```

Comment:
Should we look wether `!std::empty(base)`, or no need?

Reply:
e07e98a